### PR TITLE
Npm run setup failing in client #1219

### LINKED ISSUE #1219: `npm run setup` failing in client
@ardada2468 noted that setup is failing due to the postinstall (more details [here](https://github.com/hackforla/311-data/pull/1207#pullrequestreview-985535659)). I dug into it, and I think we can just remove the postinstall ([background](https://github.com/hackforla/311-data/pull/1207#issuecomment-1137923083)).

### PR DESCRIPTION
**Pull request by ardada2468**
Fixes #1219 

Instead of removing which could cause something to break when deploying or running,  I simply added instructions to the readme file in the client dir on how to fix the error and remove the postinstall script. 

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: client/components/main/Reports.jsx [hunk 1]
```diff
@@ -8,13 +8,14 @@ const useStyles = makeStyles(theme => ({
   root: {
     height: `calc(100vh - ${theme.header.height} - ${theme.footer.height})`,
     width: '100vw',
-    backgroundColor: 'black',
+    backgroundColor: '#1a1a1a',
```
Comment (nichhk) [inline client/components/main/Reports.jsx:11]: Please revert the changes in this file.

File: client/README.md [hunk 2]
```diff
@@ -8,6 +8,10 @@
 - From this directory, `npm run setup` to install front end dependencies and create an `.env` file
 - get a [Mapbox](https://account.mapbox.com/auth/signin/) API token and add that to your `.env` file as the `MAPBOX_TOKEN`
 - get the `API_URL` from a team member and add that to your `.env` file. Alternatively, you can [bring up](https://github.com/hackforla/311-data/blob/dev/docs/server_setup.md) your own local server and use that.
+- please replace `"postinstall": "sh -c \"are-you-es5 check -r . | tail -n 2 | head -n 1 > ./node_modules_non_es5 \""` from line 67
```
Comment (nichhk) [inline client/README.md:11]: Did you see my findings in https://github.com/hackforla/311-data/pull/1207#issuecomment-1137923083? The postinstall check is not needed anymore, so I prefer to get rid of it instead of making all our contributors manually remove it every time.

**Review by nichhk — CHANGES_REQUESTED** [review]
Thanks for working on this Arnav! A couple comments.

File: client/package.json [hunk 3]
```diff
@@ -64,7 +64,7 @@
     "test": "jest --passWithNoTests",
     "predeploy": "npm run build",
     "check-env": "node ./utils/checkEnv",
-    "postinstall": "sh -c \"are-you-es5 check -r . | tail -n 2 | head -n 1 > ./node_modules_non_es5 \""
+    "postinstall": ""
```
Comment (nichhk) [inline client/package.json:67]: We can remove line 67 entirely.

File: client/README.md [hunk 4]
```diff
@@ -9,6 +9,7 @@
 - get a [Mapbox](https://account.mapbox.com/auth/signin/) API token and add that to your `.env` file as the `MAPBOX_TOKEN`
 - get the `API_URL` from a team member and add that to your `.env` file. Alternatively, you can [bring up](https://github.com/hackforla/311-data/blob/dev/docs/server_setup.md) your own local server and use that.
 
+
```
Comment (nichhk) [inline client/README.md:12]: Please revert this change.

File: client/components/main/Reports.jsx (hunk 1 above)
Comment (nichhk) [reply client/components/main/Reports.jsx:11]: Leaving a comment again for visibility.

**Review by nichhk — CHANGES_REQUESTED** [review]

**Comment by ardada2468** [comment]
I fixed all necessary items that you have highlighted above.  

- The reason for why 3 commits did not pass the check, is becuase when removing the post install script, i forgot to remove the ',' from the script on top. 
- That issue has been patched by the last commit. 
- The background style as been revered to 'black' 
- The extra line in readme has been removed.

**Review by nichhk — CHANGES_REQUESTED** [review]
Hey Arnav, I see that you removed the postinstall line, but I'm still getting a newline in README and a color change in Reports.jsx.

Before you send a PR out for review, please check the "[Files changed](https://github.com/hackforla/311-data/pull/1290/files)" section of the PR to ensure that the changes in the PR are what you want.

Also, you don't need to manually write out whether you resolved my comments. You can use the "Resolve conversation" button on each comment to do that. Please resolve comments as you address them, otherwise GitHub won't let you submit the PR.

**Comment by nichhk** [comment]
Closed in favor of #1314.
